src/renderer/components/+welcome/welcome.tsx

```diff
@@ -34,10 +34,11 @@ export class Welcome extends React.Component {
         <div className="box">
           <Icon svg="logo-lens" className="logo" />
 
-          <h2>Welcome to {productName} 5 Beta!</h2>
+          <h2>Welcome to {productName} 5!</h2>
```

Comment:
This shouldn't be static, it should grab the version from the package.json

Reply:
lets do this in a follow up PR... This is just for UX